### ISSUE TITLE
Edit-mode Bugs and issue

### ISSUE THREAD

**Issue by joao7yt** [ThoNohT/NohBoard#34]
See the examples here: https://youtu.be/rfPb2mfPP3U

[Fixed] Bug 1: Sometimes a specific key disappears and then an error code appears when trying to left click anywhere. I don't have a clue what is causing it, seems to be random because in different keyboard definitions it behaves with different keys, but these keys are always the same on that keyboard definition. I tried to see in the .jsons but there doesn't seem to be related to the position of the keys in the json. P.S.: look that in 1:33 in the video that the "c" button, for example, doesn't have the bug... pretty crazy

[Fixed] Bug 2: When loading another keyboard while editing a key, makes it weirdly behaves. Could be prevented if you grey out the "load keyboard" when in the editmode, or stoping the edit when clicking on "load keyboard".

[Fixed] Issue 1: I can't select the entire key when it's too small because the boundaries are in front of the hitbox, and I think it should be possible, otherwise it would limit the creativity. It could be solved by making the boundaries' hitbox to be rendered outlining the button instead of over the limits. Then you could select the button area anywhere and also the bondaries, doesn't matter how small they are.

**Comment by joao7yt** [comment]
Noh, you said on stream you fixed bug number 1. I tested it now and it still there. I rewatched the part of the stream you were looking at my video example and you missed the begging of it, you missed the steps to reproduces the bug. And i also beleave you misunderstand the bug 1, it make keys disapear and crashes NohBoard, pls, look into it again :). And also in the video there's an example of all the 3 bugs/issue. 0:00 is bug 1, 2:47 is bug 2 and 3:17 is the issue. You already solved the issue, tho.
